api/hypershift/v1beta1/hostedcluster_types.go

```diff
@@ -1938,9 +1938,15 @@ type KMSSpec struct {
 
 // AzureKMSSpec defines metadata about the configuration of the Azure KMS Secret Encryption provider using Azure key vault
 type AzureKMSSpec struct {
```

Comment:
Do we need additional CEL here? Is this required?

Reply:
yes, add required validation.